lib/storage/providers/LocalForage.js

```diff
@@ -103,6 +107,14 @@ const provider = {
      */
     removeItem: localforage.removeItem,
 
+    /**
+     * Remove given keys and their values from storage
+     *
+     * @param {Array} keys
+     * @returns {Promise}
+     */
+    removeItems: keys => Promise.all(_.map(keys, key => localforage.removeItem(key))),
```

Comment:
Because you added the `removeItems` plugin, you should be able to use it here:

```suggestion
    removeItems: localforage.removeItems,
```